weekly_report/generate_report.py

```diff
@@ -1,3 +1,499 @@
+import os
+from dotenv import load_dotenv
+from reportlab.lib.pagesizes import letter
+from reportlab.pdfgen import canvas
+from reportlab.pdfbase import pdfmetrics
+from reportlab.pdfbase.ttfonts import TTFont
+from psycopg2 import connect
+from psycopg2.extras import RealDictCursor
+from psycopg2.extensions import connection
+import altair as alt
+import pandas as pd
+
+
+class Stats_Retriever():
+    """This class is responsible for the functionality which allows the user
+    to retrieve useful stats about the games database. """
+
+    def __init__(self, config=None, website_id=(1, 2, 3)) -> None:
```

Comment:
Might be useful to specify a type hint for `website_id`

Reply:
Will add!